content-src/components/ASRouterAdmin/ASRouterAdmin.jsx

```diff
@@ -229,23 +272,43 @@ export class DiscoveryStreamAdmin extends React.PureComponent {
   }
 
   render() {
+    const prefToggles = "enabled hardcoded_layout show_spocs personalized collapsible".split(
+      " "
+    );
     const { config, lastUpdated, layout } = this.props.state;
     return (
       <div>
-        <div className="dsEnabled">
-          <input
-            type="checkbox"
-            checked={config.enabled}
-            onChange={this.onEnableToggle}
-          />{" "}
-          enabled{" "}
-        </div>
+        <button className="button" onClick={this.restorePrefDefaults}>
+          Restore Pref Defaults
+        </button>{" "}
+        <button className="button" onClick={this.expireCache}>
+          Expire Cache
+        </button>
```

Comment:
It will be good if get some CTA feedback after click of Restore Defaults and Expire Cache may be a label below or next to Discovery Stream that shows action completed

Reply:
I'm mostly just following the pattern used in about:newtab#devtools-targeting and about:newtab#devtools

Those links have similar buttons.

I agree though, but probably not worth doing now.